src/parser/jobs/mnk/modules/Steppies.tsx

```diff
@@ -100,15 +107,9 @@ export default class Steppies extends Module {
 		}))
 	}
 
-	getUnbuffedCount(boots: Boot[]): number {
-		return boots.reduce((total, current) => current.weak ? total : total+1, 0)
-	}
+	getUnbuffedCount = (boots: Boot[]): number => boots.filter(boot => boot.weak).length
```

Comment:
This seems to have just been when I create, I'm flagging weak as "has Leaden Fist" instead of not having it. Filtering on `boot.weak` is correct, just the value on init was switched.